lib/stitches/cache_injector.rb

```diff
@@ -0,0 +1,32 @@
+require 'lru_redux'
+
+module Stitches::CacheInjector
+  MAX_ITEMS_IN_CACHE = ENV.fetch("API_KEY_CACHE_MAX_ITEMS", 100)
+  CACHE_DURATION_IN_SECONDS = ENV.fetch("API_KEY_CACHE_DURATION", 5 * 60) # five minutes
```

Comment:
Thanks for making this configurable!

I noticed there is a [`Stitches::Configuration`](https://github.com/stitchfix/stitches/blob/master/lib/stitches/configuration.rb) object.  I don't have a strong opinion on this, but it might be nice to consolidate all configuration of `stitches` into that object.

Reply:
My knee-jerk thought was that making this configurable was overkill (we've used this 5 minute setting in Stitch Fix apps for over a year, it's successful enough I don't think there is much reason to tweak it).

But this is a pubic app, so I think there's an argument for being sensitive to other operational concerns. I could imagine a system architected with many more valid api keys than we use, for example. So 👍  for the configurability and using Stitches::Configuration probably also makes sense. 😀